src/components/CategoryButton.js

```diff
@@ -22,26 +22,29 @@ const CategoryField = styled.section`
   display: flex;
   justify-content: space-between;
   align-items: center;
-  height: 14vh;
+  height: 17vh;
   background-color: black;
-  margin: 2vh 3vh 1vh 3vh;
+  margin: 4vh 3vh 4vh 3vh;
 
-  #LineChart {
-    width: 50px;
-  }
-
-  button {
+  .LinkContainer {
     display: flex;
     justify-content: center;
     align-items: center;
     width: 10vh;
-    height: 100%;
+    height: 10vh;
+    margin: 4px;
     background-color: darkred;
     border-width: 0;
```

Comment:
Can you please check if removing this CSS property has any effect?

Reply:
Yes, this property is necessary for adjusting the button field - it works fine.